doc/user_guide/eds.rst

```diff
@@ -512,11 +511,25 @@ The background can be subtracted from the X-ray intensities with the :py:meth:`~
 Quantification
 --------------
 
-.. versionadded:: 0.8
```

Comment:
I think that it is good to leave it as it was. You could add a `versionchanged` one describing what you've added for version `1.0`. See [this](http://www.sphinx-doc.org/en/stable/markup/para.html).